modulus/utils/zenith_angle.py

```diff
@@ -101,6 +105,197 @@ def cos_zenith_angle_from_timestamp(
     return _star_cos_zenith(julian_centuries, lon_rad, lat_rad)
 
 
+def irradiance(
+    t,
+    S0=1361,
+    e=0.0167,
+    perihelion_longitude=282.895,
+    mean_tropical_year=365.2422,
+    newton_iterations: int = 3,
+):
+    """The flux of solar energy in W/m2 towards Earth
+
+    The default orbital parameters are set to 2000 values.
+    Over the period of 1900-2100 this will result in an error of at most 0.02%,
+    so can be neglected for many applications.
+
+    Args:
```

Comment:
These doc strings are great!

Can the sections quickly get switched to Numpy styling?

```
Parameters
---------

Notes
-----
```

Reply:
done